### ISSUE TITLE
[Accessibility] A11y_.Net Editor: The "iFrame" in the page doesn't have a "Title" attribute which describes the iframe 

### ISSUE THREAD

**Issue by LadyNaggaga** [MicrosoftDocs/feedback#734]
Hello There, 
Could you please have a look at this accessibility bug? In regards to the .NET Online Editor 
https://github.com/dotnet/try/issues/119

**Comment by dend** [comment]
#log-suggestion

**Comment by botcrane** [comment]
🚀 **ATTENTION**: [Internal request](https://ceapex.visualstudio.com/web/wi.aspx?pcguid=7d644393-99ad-41c8-ac53-7fa79294c720&id=47429) logged.

**Comment by dend** [comment]
This bug has been resolved @LadyNaggaga 🥳
